src/main/scala/sangria/marshalling/testkit/MarshallingBehaviour.scala

```diff
@@ -171,6 +171,27 @@ trait MarshallingBehaviour {
       iu.isListNode(marshaled) should be (false)
     }
 
+    "(un)marshal map scalar values" in {
```

Comment:
This scenario ensures that the marshaller is able to create a scalar node from a map node value. And, that the unmarshaller can extract a GraphQL scalar from a scalar map node value.